galsim/deprecated/bandpass.py

```diff
@@ -39,7 +39,12 @@ def Bandpass_rdiv(self, other):
     else:
         tp = lambda w: other / self.func(w)
 
-    return galsim.Bandpass(tp, blue_limit, red_limit, _wave_list=wave_list)
+    if self.wave_factor == 10.0:
+        wave_type = 'Angstroms'
+    else:
+        wave_type = 'nm'
+    return galsim.Bandpass(tp, 'nm', blue_limit, red_limit,
```

Comment:
It should actually be 'nm' which was the former default (and that's why the old code didn't have a unit attached to it).  The unit conversions had already been taken care of properly.  The real problem is that I didn't notice that at first, so I added the if/else block which is totally unnecessary.  I'll remove it.